packages/zowe-explorer-api/src/fs/BaseProvider.ts

```diff
@@ -15,6 +15,18 @@ import * as path from "path";
 import { FsAbstractUtils } from "./utils";
 import { Gui } from "../globals/Gui";
 import { ZosEncoding } from "../tree";
+import { ErrorCorrelator, ZoweExplorerApiType } from "../utils/ErrorCorrelator";
+
+export interface HandleErrorOpts {
```

Comment:
The `fs/types` file is mostly for the file system types (data set / USS / jobs / generic entries) rather than interfaces, but I can see value in it being moved there for organization. 

I figured that this interface is only useful in the context of `BaseProvider`, so that you can grab everything you need for the base provider in one file 😋 if it doesn't cause any circular imports, I'll move it 👍